### PR TITLE
feature: pdf template custom page size.

### PR DESCRIPTION
**Pull request by ayman-alkom**
I can confirm that this change tested and works with no problems

### CODE REVIEW

**Comment by ayman-alkom** [comment]
https://youtu.be/MnFfXmxSOd4

**Comment by yurikuzn** [comment]
Since it's a feature, pr should be to master rather than hotfix. Thanks.

**Comment by yurikuzn** [comment]
If you would like me to make changes by myself please PR to the branch `feature/pdf-custom-page-size`.

File: application/Espo/Resources/metadata/clientDefs/Template.json [hunk 1]
```diff
@@ -20,6 +20,22 @@
                     "fields": ["footer", "footerPosition"]
                 }
             ]
+        },
+        "pageFormat": {
```
[the remark this answers is not in the document]
Comment (ayman-alkom) [reply application/Espo/Resources/metadata/clientDefs/Template.json]: this solved

File: application/Espo/Resources/i18n/en_US/Template.json [hunk 2]
```diff
@@ -14,6 +14,8 @@
         "variables": "Available Placeholders",
         "pageOrientation": "Page Orientation",
         "pageFormat": "Paper Format",
+        "width": "Width (cm)",
+        "height": "Height (cm)",
```
[the remark this answers is not in the document]
Comment (ayman-alkom) [reply application/Espo/Resources/i18n/en_US/Template.json]: fixed

**Comment by ayman-alkom** [comment]
> If you would like me to make changes by myself please PR to the branch `feature/pdf-custom-page-size`.

I am happy to do it by myself, no problem

File: application/Espo/Resources/i18n/en_US/Template.json [hunk 3]
```diff
@@ -14,6 +14,8 @@
...
         "fontFace": "Font"
```
[the remark this answers is not in the document]
Comment (ayman-alkom) [reply application/Espo/Resources/i18n/en_US/Template.json]: added

**Comment by ayman-alkom** [comment]
I made all changes to be in one commit

**Pull request closed without merging**